extensions/panache/rest-data-panache/deployment/src/main/java/io/quarkus/rest/data/panache/deployment/methods/StandardMethodImplementor.java

```diff
@@ -235,6 +236,11 @@ protected boolean isNotReactivePanache() {
         return !capabilities.isPresent(Capability.HIBERNATE_REACTIVE);
     }
 
+    private static Enum schemaTypeArray() {
+        Class<?> schemaTypeClass = toClass(SCHEMA_TYPE_CLASS_NAME);
+        return Enum.valueOf((Class<Enum>) schemaTypeClass, SCHEMA_TYPE_ARRAY);
```

Comment:
This can probably be made a constant, no?

Reply:
This class is optional (it's only available when the `quarkus-smallrye-openapi` extension is used). 
We're only resolving the class when this capability is enabled: https://github.com/quarkusio/quarkus/blob/0bc09e788b4f40341b98e90b89c051df1e9d936c/extensions/panache/rest-data-panache/deployment/src/main/java/io/quarkus/rest/data/panache/deployment/methods/StandardMethodImplementor.java#L196

Reply:
Okay, no problem